src/widgets/DataTable/DataTableHeaderRow.js

```diff
@@ -41,7 +41,7 @@ const DataTableHeaderRow = React.memo(({ columns, sortKey, isAscending, onPress
             columnKey={key}
             onPress={sortable ? onPress : null}
             sortDirection={directionForThisColumn}
-            sortable={sortable}
+            sortable={isSortable ? sortable : false}
```

Comment:
the columns of the patient select weren't originally sortable, and I didn't want to bother spending time to do it